crates/uv/src/main.rs

```diff
@@ -439,6 +439,10 @@ struct PipSyncArgs {
     #[clap(long, conflicts_with = "index_url", conflicts_with = "extra_index_url")]
     no_index: bool,
 
+    /// The Python interpreter into which to install the packages.
```

Comment:
It feels weird to consider installing packages into an interpreter. Perhaps we should rephrase?

Separately, "into which to" is really confusing

Reply:
Focused on other things right now, I'll come back to this comment later.